exporter/cdn_exporter.go

```diff
@@ -93,73 +95,75 @@ func CdnCloudExporter(c *cms.Client) *CdnExporter {
 
 //导出
 func (e *CdnExporter) Describe(ch chan<- *prometheus.Desc) {
-	ch <- e.cdnhitRate
-	ch <- e.cdnoriBps
-	ch <- e.cdnl1Acc
-	ch <- e.cdnoriAcc
-	ch <- e.cdnBPS
-	ch <- e.cdnoricode4xxratio
-	ch <- e.cdnoricode5xxratio
+	ch <- e.hitRate
+	ch <- e.backSourceBps
+	ch <- e.BPS
+	ch <- e.backSourceAcc
+	ch <- e.l1Acc
+	ch <- e.backSourceStatusRatio
+	ch <- e.StatusRatio
 }
 
 //收集
 func (e *CdnExporter) Collect(ch chan<- prometheus.Metric) {
 	cdnDashboard := collector.NewCdnExporter(e.client)
 
-	for _, point := range cdnDashboard.RetrievehitRate() {
+	for _, point := range cdnDashboard.RetrieveHitRate() {
 		ch <- prometheus.MustNewConstMetric(
-			e.cdnhitRate,
+			e.hitRate,
 			prometheus.GaugeValue,
 			float64(point.Average),
 			point.InstanceId,
 		)
 	}
-	for _, point := range cdnDashboard.RetrieveoriBps() {
+	for _, point := range cdnDashboard.RetrieveOriBps() {
 		ch <- prometheus.MustNewConstMetric(
-			e.cdnoriBps,
+			e.backSourceBps,
 			prometheus.GaugeValue,
-			float64(point.Average),
+			float64(point.Average / 1000000),
```

Comment:
```suggestion
			float64(point.Average / 1024 / 1024 ),
```
确认一下 阿里云的计算方法, 不一定是 / 1000, 不要和阿里云的dashboard 上的数据有差别

Reply:
![image](https://user-images.githubusercontent.com/51444798/188564873-dd832eb0-eaef-4016-ac94-0cbd2ec9e16f.png)

Reply:
```suggestion
			float64(point.Average / 1000 / 1000),
```